### DISCUSSION TITLE
Feature request to be able to exclude subsections of a method

### DISCUSSION THREAD

**Discussion by bl-young** [USEPA/flowsa#256] [Ideas]
Currently we can include methods from other yaml files like `!include:GHG_national_2016_m1.yaml:source_names:EPA_GHGI_T_2_1`

In a few cases, it would be nice to pull in a whole source BUT exclude one piece (like an activity set). Is this possible?

**Comment by matthewlchambers** [comment]
I agree this would be very useful. Currently, we can override arbitrary keys from the `!include:`-ed yaml file, but this does not cover all use cases. The `yaml` specification only allows one tag per key, otherwise I would say we should implement Ben's proposal for an `!exclude:` tag which could follow `!include:`. As it is, we'll have to consider the best way to implement.
